components/molecules/FeaturedHighlightsPanel/featured-highlights-panel.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from "next/link";
+import React, { useEffect, useState } from "react";
+
+interface FeaturedHighlightsPanelProps {
+  highlightIds: { id: number; title: string }[];
+}
+const FeaturedHighlightsPanel = ({ highlightIds }: FeaturedHighlightsPanelProps) => {
```

Comment:
```suggestion
const FeaturedHighlightsPanel = ({ highlightsData }: FeaturedHighlightsPanelProps) => {
```
When I read this code I thought at some array of number